proto/one/enterprise/data/enterprise_dataset_definition.proto

```diff
@@ -16,11 +16,14 @@ message EnterpriseDatasetDefinition {
     // Version of dataset
     string version = 3;
 
+    // Dataset belonging tenant twin id
+    string TenantId = 4;
```

Comment:
This is probably ok since we are not using this one yet but typically we do not renumber fields

Reply:
I am not planning bump up c# model library version as well, I am the only one using it.